ci-operator/step-registry/ipi/install/hosted-loki/ipi-install-hosted-loki-commands.sh

```diff
@@ -550,3 +550,6 @@ metadata:
   name: loki-promtail
   namespace: loki
 EOF
+
+
+echo "Promtail manifests created, the cluster can be found at https://grafana-loki.ci.openshift.org/explore using '{invoker=\"${OPENSHIFT_INSTALL_INVOKER}\"}' query"
```

Comment:
Why not use the completed URL that includes the query:
```
https://grafana-loki.ci.openshift.org/explore?orgId=1&left=%5B%22now-1h%22,%22now%22,%22Grafana%20Cloud%22,%7B%22expr%22:%22%7Binvoker%3D%5C%22openshift-internal-ci%2Fe2e-agnostic%2F1364498370567081984%5C%22%7D%22%7D%5D
```
You would need a urlencode call.

Reply:
^ this makes sense to me, but can be a follow-up improvement.  This PR is already a big enough improvement over "no advice" that I'd rather not block on a completed URI.

/lgtm